umh.docs.umh.app/content/en/docs/features/connectivity/additionalconnectivity/barcodereader-retrofitting.md

```diff
@@ -25,13 +25,24 @@ something like a quality testing station (we once connected a Mitutoyo quality t
 
 To use the microservice barcode reader, you will need configure the helm-chart and enable it.
 
-1. Enable \_000_commonConfig.datasources.barcodereader.enabled in the [Helm Chart](https://umh.docs.umh.app/docs/architecture/helm-chart/#configuration-options)
+1. Enable the barcodereader feature by executing the following command:
+```bash
+sudo $(which helm) upgrade --kubeconfig /etc/rancher/k3s/k3s.yaml  -n united-manufacturing-hub united-manufacturing-hub united-manufacturing-hub/united-manufacturing-hub --set _000_commonConfig.datasources.barcodereader.enabled=true --reuse-values --version $(sudo $(which helm) l
+s --kubeconfig /etc/rancher/k3s/k3s.yaml  -n united-manufacturing-hub -o json | jq -r '.[0].app_version')
+```
```

Comment:
Indent the code block to be in line with the first word of the list (in this case, with "Enable")